src/factors/PartialPose3.jl

```diff
@@ -19,11 +19,23 @@ end
 
 PriorPose3ZRP(z::T1,rp::T2) where {T1 <: SamplableBelief, T2 <: SamplableBelief} = PriorPose3ZRP{T1,T2}(z, rp)
 
-getManifold(::PriorPose3ZRP) = ProductGroup(ProductManifold(TranslationGroup(1), SpecialOrthogonal(3)))
+getManifold(::PriorPose3ZRP) = SpecialEuclidean(3)
 
-#FIXME update
-function getSample(cfo::CalcFactor{<:PriorPose3ZRP})
-  return ([rand(cfo.factor.z); rand(cfo.factor.rp)], )
+#FIXME update to also only one measurement
+function getSample(cf::CalcFactor{<:PriorPose3ZRP})
+  
+  M = getManifold(cf.factor)
+
+  ϵ = identity_element(M)
+
+  #Rotation part: roll and pitch
+  r,p = rand(cf.factor.rp)
+  R = Rotations.RotXY(r, p)
+  
+  # Translation part: Z
+  T = [0; 0; rand(cf.factor.z)]
+
+  return (ProductRepr(T, R), )
```

Comment:
just for parity -- returning a `::Tuple` of manifold point because this is a `<:AbstractPrior`.  

For `<:AbstractRelative` we will return `::Tuple` with manifold tangents about some point (how and where the point is specified is up to the user, since the user implements both the `getSample` and 'residual' functions.

Reply:
Yes, Priors should currently return points on manifold